### PR TITLE
Add code coverage badge

### PR DESCRIPTION
**Pull request by zeroual**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # junit5-docker
 
 [![Build Status](https://travis-ci.org/FaustXVI/junit5-docker.svg?branch=master)](https://travis-ci.org/FaustXVI/junit5-docker)
+[![codecov](https://codecov.io/gh/zeroual/junit5-docker/branch/master/graph/badge.svg)](https://codecov.io/gh/zeroual/junit5-docker)
```
Comment (vdemeester) [inline README.md:4]: You will have to use `FaustXVI` url here :angel:

Comment (zeroual) [reply README.md:4]: yeap , i asked @FaustXVI  to create an account and give me the new url

Comment (FaustXVI) [reply README.md:4]: https://codecov.io/gh/FaustXVI/junit5-docker here it is :)

**Review by FaustXVI — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ script: mvn verify
 after_success:
     - .travis/publish-documentation.sh
     - .travis/deploy-snapshot
+    - bash <(curl -s https://codecov.io/bash)
```
Comment (FaustXVI) [inline .travis.yml:11]: mvn jacoco:report will be needed too :)

**Comment by vdemeester** [comment]
ping @zeroual

**Comment by FaustXVI** [comment]
@zeroual should we close the PR?

**Pull request closed without merging**